qiling/os/posix/syscall/unistd.py

```diff
@@ -143,12 +143,14 @@ def ql_syscall_faccessat(ql: Qiling, dfd: int, filename: int, mode: int):
 
     if not os.path.exists(real_path):
         regreturn = -1
-
-    elif stat.S_ISFIFO(Stat(real_path).st_mode):
+    else:
         regreturn = 0
+    # TODO: according to real implementation, idk why here was a FIFO check
```

Comment:
I track the history for this code. Looks like it's introduced here: https://github.com/qilingframework/qiling/commit/2f6ff4383ef7ae9b1707bdc5313f957849431ef3. Thus, it should be safe to remove the comments totatly.